### PR TITLE
otk: generalize include for non dict values

### PR DESCRIPTION
**Pull request by mvo5**
Small followup for PR#141, deal with non dict types from includes in a more generic way. Includes can also just include vales.

### CODE REVIEW

**Review by supakeen — DISMISSED** [review]
Sure, this works too. I'd prefer it if we are even more explicit with an `isinstance(..., dict) elif isinstance(..., list) elif raise` style?

Perhaps that's too much LBYL :)

File: test/data/base/12-op-join-type.yaml [hunk 1]
```diff
@@ -7,3 +7,5 @@ otk.target.osbuild.test:
         - - 1
           - 2
         - otk.include: "12-op-join-type/example.yaml"
+        - - otk.include: "12-op-join-type/val.yaml"
+        - - 6
```
[the remark this answers is not in the document]
Comment (schuellerf) [reply test/data/base/12-op-join-type.yaml:11]: other than that there is only floating point number missing, I guess?

[the remark this answers is not in the document]
Comment (schuellerf) [reply test/data/base/12-op-join-type.yaml:11]: oh - and booleans, but then it's complete :-D

[the remark this answers is not in the document]
Comment (mvo5) [reply test/data/base/12-op-join-type.yaml:14]: Thank you! Nice suggestion, I added a commit for it and added you as co-author (couldn't just apply the suggestion verbatim as GH does not seem to support adding new files this way).

**Comment by mvo5** [comment]
> Sure, this works too. I'd prefer it if we are even more explicit with an `isinstance(..., dict) elif isinstance(..., list) elif raise` style?
> 
> Perhaps that's too much LBYL :)

I tried this idea and it becomes a relatively long line/block which seems harder to read - but maybe I'm missing something? I wonder if a comment instead would be better(?)

**Review by supakeen — APPROVED** [review]

**Pull request merged**